portfolio.js

```diff
@@ -0,0 +1,28 @@
+const hambergur = document.querySelector('.allign');
+const contactsMenu = document.querySelector('.contacts-menu');
+const closes = document.querySelector('.close-menu');
+const main = document.querySelector('.main');
+const navlink = document.querySelectorAll('.navlinks');
+const navContainer = document.querySelector('.nav-container');
+hambergur.addEventListener('click', () => {
+  hambergur.classList.toggle('close');
+  closes.style.display = 'block';
+  contactsMenu.classList.toggle('active');
+  navContainer.style.height = '300px';
+  main.style.opacity = '0';
+  hambergur.style.display = 'none';
+});
+closes.addEventListener('click', () => {
+  closes.classList.toggle('allign');
+  hambergur.style.display = 'block';
+  closes.style.display = 'none';
+  navContainer.style.height = '92px';
+  contactsMenu.style.display = 'none';
+  main.style.opacity = '1';
+});
```

Comment:
### Second Time Use
![second-time](https://user-images.githubusercontent.com/93039359/179869381-3a124b1d-703c-4086-888c-d8fdfd75e06f.png)

- [ ] Once we use the menu in your portfolio then the second time it behaves something like shown in the image :up: Kindly fix it :blush: